plugins/Plugins.cmake

```diff
@@ -111,6 +111,7 @@ MACRO(DFHACK_PLUGIN)
     SET_SOURCE_FILES_PROPERTIES( ${PLUGIN_PROTO_HDRS} PROPERTIES HEADER_FILE_ONLY TRUE )
     LIST(APPEND PLUGIN_SOURCES ${PLUGIN_PROTO_HDRS})
     LIST(APPEND PLUGIN_SOURCES ${PLUGIN_PROTO_SRCS})
+    LIST(APPEND PLUGIN_SOURCES ${PLUGIN_PROTOS})
```

Comment:
Why? This is passed into ADD_LIBRARY below - wouldn't this try to compile .proto files?

Reply:
No, the compilers recognize them as not being source files, but this way they show up in the file list in visual studio as an extra file, which makes editing them together with the rest of the stuff much more convenient.

Reply:
Oh, right, like header files